packages/astro/src/default/components/MetaTags.astro

```diff
@@ -0,0 +1,36 @@
+---
+import type { MetaTagsConfig } from '@tutorialkit/types';
+import { readFaviconFile } from '../utils/favicon';
+import { readLogoFile } from '../utils/logo';
+import { readPublicImage } from '../utils/publicImage';
+
+interface Props {
+  meta?: MetaTagsConfig;
+}
+const { meta = {} } = Astro.props;
+const site = Astro.site?.toString();
+let imageUrl;
+if (meta.image) {
+  imageUrl = readPublicImage(meta.image, true);
+  if (!imageUrl) {
+    console.warn(`Image ${meta.image} not found in "/public" folder`);
+  }
+}
+imageUrl ??= readLogoFile('logo', true);
+const faviconUrl = readFaviconFile('favicon', true);
+---
+
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width" />
+<meta name="generator" content={Astro.generator} />
+{meta.description ? <meta name="description" content={meta.description} /> : null}
+{meta.title ? <title>{meta.title}</title> : null}
```

Comment:
Should `title` and `og:title` always be the same? I'm tempted to think we should keep them separate.

@AriPerkkio what do you think?

Reply:
Yeah handling the title raises some questions, title is used in the navbar and is currently computed as a breadcrumbs, while meta.title is what you want to ve visible on social media

Reply:
Setting `docment.title` from `meta.title` doesn't sound right to me. So maybe it's better to have separate ones.

Reply:
Makes sense, I'll stick to the lesson's title then for the document title